locale/zh-cn/docs/guides/event-loop-timers-and-nexttick.md

```diff
@@ -191,15 +192,15 @@ immediate
 timeout
 ```
 
-使用 `setImmediate()` 超过 `setTimeout()` 的主要优点是 `setImmediate()` 在任何计时器（如果在 I/O 周期内）都将始终执行，而不依赖于存在多少个计时器。
+使用 `setImmediate()` 相对于`setTimeout()` 的主要优势是，如果`setImmediate()`是在I/O周期内被调度的，那它将会在其中任何的定时器之前执行， 跟这里存在多少个定时器无关
```

Comment:
add space between English or special characters and Chinese characters.

https://github.com/nodejs/nodejs-zh-CN
> 请直接前往 [Issue 列表](https://github.com/nodejs/iojs-cn/issues)找到你感兴趣的 Issue 参与讨论，也可申请指派翻译任务。为了保持一致性，提升文档的质量，**在翻译之前建议先查阅下[中英文排版规范](https://github.com/sparanoid/chinese-copywriting-guidelines)**。请按照以下流程执行，避免浪费人力做相同的事情。